src/locales/en/fields.yml

```diff
@@ -1,10 +1,11 @@
 nationalInsuranceNumber:
   content: " "
-  label: What is your national insurance number?
-  hint: It's on your National Insurance card, benefit letter, payslip or P60. For example, 'QQ123456C'
+  label: "National Insurance number"
+  hint: "For example, ‘QQ 12 34 56 C’."
```

Comment:
I'm being really nitpicky here but there is a full stop in English and not Welsh, can we align them?